libs/cdk/package.json

```diff
@@ -48,7 +48,8 @@
     "url": "https://github.com/rx-angular/rx-angular.git"
   },
   "peerDependencies": {
-    "@angular/core": ">10.1.1"
+    "@angular/core": ">=8.0.0",
```

Comment:
do we really support angular 8?

Reply:
Not tested, but I guess so. It's not very clear which version we support.